sys/usb/usbus/cdc/acm/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_USBUS_CDC_ACM
+    bool "Configure USBUS CDC ACM"
+    depends on MODULE_USBUS_CDC_ACM
+    help
+        Configure the USBUS CDC ACM module via Kconfig.
+
+if KCONFIG_MODULE_USBUS_CDC_ACM
+
+config USBUS_CDC_ACM_STDIO_BUF_SIZE
+    int "Buffer size for STDIN and STDOUT data"
```

Comment:
This could use a small note in the help that this is only used with the CDC ACM STDIO module and not with `USBUS_CDC_ACM` (and maybe also guards to only show this with `STDIO_CDC_ACM` enabled, but I'll leave that up to you to decide)

Reply:
This already depends on `MODULE_STDIO_CDC_ACM`, so the option should not show up unless the module is active

Reply:
Whoops, should have checked this better!